templates/footer.html

```diff
@@ -1,16 +1,17 @@
 <footer>
-  <p>Except as otherwise noted, the content of this page under <a href="https://creativecommons.org/licenses/by/2.5/">CC Attribution 2.5</a> license. Code examples are <a href="https://github.com/GoogleChrome/samples/blob/gh-pages/LICENSE">Apache-2.0</a>.</p>
   <div>
-    <a href="https://groups.google.com/a/chromium.org/forum/#!newtopic/blink-dev"
-       >File content issue</a> |
-    <a href="https://github.com/GoogleChrome/chromium-dashboard/issues"
-       >File an issue</a> |
+     <a href="https://groups.google.com/a/chromium.org/forum/#!newtopic/blink-dev"
```

Comment:
Is it intended to have some links target="_blank", some not?

Reply:
Added.